main.go

```diff
@@ -130,20 +130,36 @@ func processGeofeed(geofeedFilename, mmdbFilename string) (counts, []string, err
 	csvReader := csv.NewReader(geofeedFH)
 	csvReader.ReuseRecord = true
 	csvReader.Comment = '#'
-	csvReader.FieldsPerRecord = 5
+	csvReader.FieldsPerRecord = -1
 	csvReader.TrimLeadingSpace = true
 
+	const expectedFieldsPerRecord = 5
+
+	// We will assume one record per line. Multi-line records are valid
+	// so this assumption isn't guaranteed to hold up; but with only five
+	// fields per record, asking the geofeed producer to restrict themselves
+	// to one record per line seems reasonable.
```

Comment:
I'm unsure if this comment is needed, and here is why:

Reading the linked RFC, we see

From "2.1 Specification":
>   Feed lines that are not comments MUST be formatted as comma-separated
>   values (CSV), as described in [RFC4180].  Each feed entry is a text
>   line of the form:
>   ip_prefix,alpha2code,region,city,postal_code

and less conclusively, from "2.0 Self-Published IP Geolocation Feeds":

>   At its simplest, the mechanism consists of a network operator
>   publishing a file (the "geolocation feed") that contains several text
>   entries, one per line.

What do you think?